FirebaseFirestoreTestingSupport.podspec

```diff
@@ -0,0 +1,63 @@
+Pod::Spec.new do |s|
+  s.name                    = 'FirebaseFirestoreTestingSupport'
```

Comment:
Should we
a) define a pod for each of the products we support (i.e. FirebaseFirestoreTestingSupport, FirebaseAuthTestingSupport, etc.)
b) define a single pod (FirebaseTestingSupport)

Reply:
Good question! Initially I wanted to have a single FirebaseTestingSupport spec but discovered the following disadvantages:
- with a single podspec we will have to add unnecessary dependencies to the test targets (e.g. we probably don't want adding Firestore dependency to Auth tests)
- OS versions requirements may be different for different SDKs. It's harder to express with a single FirebaseTestingSupport spec.

So my proposal is to have separate podspecs/SPM targets for testing support code per SDK.

WDYT?

Reply:
SG - happy with the extra flexibility this gives us.